packages/server/server/api/destinations/destination.interface.ts

```diff
@@ -0,0 +1,24 @@
+export enum DestinationCategory {
+    FOOD = 'food',
+    ACCOMMODATION = 'accommodation',
+    ENTERTAINMENT = 'entertainment',
```

Comment:
There is a new category? Time to go back to the spreadsheet and reorganize 😅 And it seems like we will be storing multiple categories for each destination, but how do we choose which category icon to display? randomly?

Reply:
NOooo sorry, my bad. You don't need to change it, we can just stick to what the front end has. We store multiple categories, but I think we can just show arbitrarily the first one in the list for now?

Reply:
okayy, entertainment is a good addition though lol. 
yea as long as it is consistent, we should be good. we dun want users to get different category for the same destination every time they login haha